src/controllers/update/check.your.answers.controller.ts

```diff
@@ -30,7 +30,6 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
 
     const changeLinkUrl: string = OVERSEAS_ENTITY_UPDATE_DETAILS_URL;
     const overseasEntityHeading: string = OVERSEAS_ENTITY_SECTION_HEADING;
-
```

Comment:
maybe undo this unless intended? 
I just like to keep file changes to as minimum as possible but up to you

Reply:
restored